doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -556,6 +560,9 @@ Bluetooth libraries and services
   * Updated the :c:func:`bt_hogp_rep_read` function to forward the GATT read error code through the registered user callback.
     This ensures that API user is aware of the error.
 
+* :ref:`bt_fast_pair_readme` library:
```

Comment:
the entries should be moved to the release notes PR, as it's already open

Reply:
To this one https://github.com/nrfconnect/sdk-nrf/pull/19241, please.

Reply:
Yes, I want to get approves here and then I will move it to the release notes PR.

Reply:
The https://github.com/nrfconnect/sdk-nrf/pull/19241 is merged now. Hence, we could keep this entry in the PR itself.